client/my-sites/media/controller.js

```diff
@@ -36,7 +43,8 @@ module.exports = {
 		// Render
 		renderWithReduxStore(
 			React.createElement( MediaComponent, {
-				sites: sites,
+				selectedSite,
+				selectedSiteSlug,
```

Comment:
There's some corner cases for Jetpack sites that we should be using the selector for